Detectors/TPC/base/include/TPCBase/ParameterElectronics.h

```diff
@@ -16,8 +16,8 @@
 #define ALICEO2_TPC_ParameterElectronics_H_
 
 #include <array>
-#include "SimConfig/ConfigurableParam.h"
-#include "SimConfig/ConfigurableParamHelper.h"
+#include "CommonUtils/ConfigurableParam.h"
```

Comment:
If you want to make this a common utility and not one only for simulation, I think it should be combined with the ConfigParamSpec facility which is already there. What is missing from the latter to be used?

Reply:
No idea (I think ConfigParamSpec is for pure command line options). I need to move this class because there was a cycling dependency. We can talk about a generalization later independent of this PR.

Reply:
> We can talk about a generalization later independent of this PR

Yes, sure. ConfigParamSpec is not for pure command line options, btw.